api/python/provisioner/commands/upgrade/set_swupgrade_repo.py

```diff
@@ -50,6 +50,8 @@
                          ReleaseInfoValidator,
                          YumRepoDataValidator,
                          HashSumValidator)
+from ...vendor import attr
```

Comment:
please use absolute imports, three dots is too far

Reply:
Done